### PR TITLE
added commenting from disqus to show ease of implementation. For reference implementation. 

### PR DESCRIPTION
**Pull request by HN23**

### CODE REVIEW

**Comment by HN23** [comment]
comments are page specific. It has a comment box for all channels, posts, etc though. should it?

**Comment by SeanHayes** [comment]
Could you instead override the mesh/base.html template and put the Disqus code there? That'll limit Disqus comments to django-mesh content, instead of putting comments everywhere.

Also, don't forget about adding that setting to our project settings.

**Comment by SeanHayes** [comment]
https://github.com/mfogel/django-settings-context-processor

File: django_mesh/templates/django_mesh/base.html [hunk 1]
```diff
@@ -1 +1,16 @@
 {% extends "base.html" %}
+{% block comment %}
```
Comment (SeanHayes) [inline django_mesh/templates/django_mesh/base.html]: Could you override this file in:
django_mesh_test_project/templates/django_mesh/base.html

We want to show an example of using Disqus but not require it in the importable module.

File: django_mesh_test_project/templates/base.html [hunk 2]
```diff
@@ -146,6 +140,13 @@
...
 		</div>
-
 	{% endblock body %}</body>
-</html>
\ No newline at end of file
+
+</html>
+
+		<!-- JS -->
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (SeanHayes) [inline django_mesh_test_project/templates/base.html]: These lines should go just inside the end of the <body>.

File: django_mesh_test_project/templates/base.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	{%load static%}
 
 <html xmlns="http://www.w3.org/1999/xhtml"  xmlns:og="http://opengraphprotocol.org/schema/" xmlns:fb="http://www.facebook.com/2008/fbml" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}" xml:lang="{{ LANGUAGE_CODE }}" lang="{{ LANGUAGE_CODE }}">
-	<head>
```
Comment (SeanHayes) [inline django_mesh_test_project/templates/base.html]: I think this was accidentally removed.

File: setup.py [hunk 4]
```diff
@@ -47,7 +47,7 @@ def runtests():
         'django_mesh_test_project',
     ],
     include_package_data=True,
-    install_requires=['Django>=1.7', 'django-model-utils',],
+    install_requires=['Django>=1.7', 'django-model-utils', 'django-settings-context-processor',],
```
Comment (SeanHayes) [inline setup.py]: Could you move this to "tests_require"? It's needed by our tests but won't actually be required to install.

File: README.rst [hunk 5]
```diff
@@ -13,10 +13,10 @@ Depends on
 ----------
 
 * Django 1.7
- * auth, admin, comments
+* auth, admin, comments
 * django-model-utils
 * python-pytz
-
+* django-settings-context-processor
```
Comment (SeanHayes) [inline README.rst]: This is needed by tests but not needed to use the app itself.

**Pull request merged**